core/Helper/Data.php

```diff
@@ -350,12 +352,37 @@ public function getFormKey()
         return $this->formKey->getFormKey();
     }
 
+    /**
+     * Get the default first name for all dealers
+     * @return string
+     */
+    public function getDefaultFirstName()
+    {
+        $quote = $this->checkoutSession->getQuote();
+        $customer = $quote->getCustomer();
+
+        if ($customer && $customer->getId()) {
+            $firstName = $customer->getFirstname();
+            
+            return $firstName;
+        }
+        return self::DEFAULT_FIRSTNAME;
+    }
+
     /**
      * Get the default last name for all dealers
      * @return string
      */
     public function getDefaultLastName()
     {
+        $quote = $this->checkoutSession->getQuote();
+        $customer = $quote->getCustomer();
+
+        if ($customer && $customer->getId()) {
+            $lastname = $customer->getLastname();
```

Comment:
Same here. You can `return $customer->getLastname();` instead of assigning the value to $lastname